dependencies.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
     compileOnly('com.github.GTNewHorizons:ForgeMultipart:1.2.7:dev') {transitive=false}
     compileOnly('com.github.GTNewHorizons:NotEnoughItems:2.2.7-GTNH:dev')
     compileOnly('com.github.GTNewHorizons:BuildCraft:7.1.27:api') {transitive=false}
+    compile('com.github.GTNewHorizons:GTNHLib:0.0.4:dev')
```

Comment:
You're adding an entire library for just one class, which even looks like you added it to GTNHLib for the sole purpose of using it here.

Please just add that code directly in here instead of adding dependency baggage.

Reply:
It was requested by someone else to add it to GTNHlib (which is reusable stuff like this) instead of directly here...  GTNHlib has no transitive deps, so depending on it is fine.

Reply:
Adding GTNHLib is totally fine. No reason to copy over code.